### PR TITLE
Uploaded in Assignment_7

### PR DESCRIPTION
**Pull request by NikitaShivhare1**

### CODE REVIEW

File: Nikita/Assignment_7/academic.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <div class="center">
+        <div class="Academic">
+            <h2> Academic Details </h2>
+            <form id="Academic" method="Post"></form>
+<form class="form2">
+    <fieldset>
+        <label class="label-checkbox1"><b>Skill Set:</b></label>
+        <input type="checkbox" name="g" value="html" />HTML
+        <input type="checkbox" name="g" value="java_script" />Java
+        Script <input type="checkbox" name="g" value="java" />Java
```
Comment (shubhamdalal49) [inline Nikita/Assignment_7/academic.html]: check indentation

File: Nikita/Assignment_7/academic.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                <option value="Pune University"></option>
+                <option value="Chennai University"></option>
+                <option value="Nagpur University"></option>
+                <option value="Badoda University"></option>
+            </datalist>
+        </div>
+        <div class="multipleoption1">
+            <label class="label-multiple"
+                ><b>Areas of Expertise:</b></label
+            >
```
Comment (shubhamdalal49) [inline Nikita/Assignment_7/academic.html]: closing tag sign is inline make them changes

File: Nikita/Assignment_7/academic.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                <option>Software</option>
+                <option>Training</option>
+                <option>Networking</option>
+                <option>Managrment</option>
+                <option>Maeketing</option>
+            </select>
+            <simpletext>You can select multiple fields.</simpletext>
+        </div>
+    </fieldset>
+    <br />
```
Comment (shubhamdalal49) [inline Nikita/Assignment_7/academic.html]: add one line

File: Nikita/Assignment_7/login.css  [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    text-align: center;
+}
+body {
+    background-image: url("Images/4.jpg");
+    background-size: cover;
+    opacity: 0.9;
+}
+form {
+    font-family: Georgia, "Times New Roman", Times, serif;
+}
```
Comment (shubhamdalal49) [inline Nikita/Assignment_7/login.css ]: add one line at the end

File: Nikita/Assignment_7/login.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+                            <div class="form-group"><input type="button" value="Submit" class="form-control btn-primary" />
+                                <p style="color:chartreuse; font-size: medium; text-align: center;margin-top: 5px;">For New User Create an Account</p></div>
+                        </form>
+                    </div>
+                </div>
+                <div class="col-md-4"></div>
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (shubhamdalal49) [inline Nikita/Assignment_7/login.html]: add one line at the end

File: Nikita/Assignment_7/persional.html [hunk 6]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="persional.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Personal Details </title>
+</head>
+<body>
```
Comment (shubhamdalal49) [inline Nikita/Assignment_7/persional.html]: check indentation

File: Nikita/Assignment_7/persional.html [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+            &nbsp;&nbsp;&nbsp;&nbsp;
+            <input type="radio" name=" gender"
+            id="female">
+            &nbsp;
+            <span id="female"> Female </span>
+            <br><br>
+            <input type="submit" value="Submit"
+            name="submit" id="submit">
+        </form>
+        </div>
```
Comment (shubhamdalal49) [inline Nikita/Assignment_7/persional.html]: check indentation

File: Nikita/Assignment_7/persional.html [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+    </div>
+</body>
+</html>
```
Comment (shubhamdalal49) [inline Nikita/Assignment_7/persional.html]: add extra line at the end

File: Nikita/Assignment_8/ass_8.css [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+    float: right;
+    width: 500px;
+    height: 1500px;
+    padding: 10px;
+}
+text{
+    text-align: left;
+    font-family: verdana;
+    height: 200px;
+
```
Comment (shubhamdalal49) [inline Nikita/Assignment_8/ass_8.css]: remove this space

File: Nikita/Assignment_8/ass_8.css [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+    float: left;
+    width: 500px;
+    height: 150px;
+    padding: 10px;
+}
+text{
+    text-align: right;
+    font-family: verdana;
+    height: 200px;
+}
```
Comment (shubhamdalal49) [inline Nikita/Assignment_8/ass_8.css]: add extra line at the end

File: Nikita/Assignment_8/ass_8.html [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+            <img src="home.jpg"
+            alt="Norway" style="width:20%; float: right;" >
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Perspiciatis eos reprehenderit modi incidunt, eum explicabo hic itaque consequuntur omnis ullam quam quae, doloribus quas tempora ipsam quo, voluptas dolorem? Saepe dolor quos ut repellendus laudantium id placeat aliquid, nam in expedita unde autem cupiditate dolorum amet iure provident, aut reiciendis optio voluptatum minima necessitatibus, adipisci quae cum natus? Iure repellendus at ullam quo fugit. In, esse. Eveniet facere repellendus ducimus, dolores cumque iste dolore nisi ad sed, officia facilis alias? Quis nisi alias exercitationem ullam, omnis beatae facilis vel aliquam in quo ipsam, doloremque officia amet quae sint consequatur suscipit!</p>
+            <img src="home.jpg" alt="Norway" style="width:20%; float: left;">
```
Comment (shubhamdalal49) [inline Nikita/Assignment_8/ass_8.html]: check indentation

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

File: Nikita/Assignment_7/login.css  (hunk 4 above)
Comment (Pawan1005) [inline Nikita/Assignment_7/login.css :28]: some thing is wrong in this files make it correct

File: Nikita/Assignment_7/persional.html (hunk 8 above)
Comment (Pawan1005) [inline Nikita/Assignment_7/persional.html]: indentation in wrong

File: Nikita/Assignment_8/ass_8.html [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+            <p> Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quia labore a, odit dolorum impedit iusto sed aliquid ullam ea adipisci officia aliquam ad eos accusamus facilis mollitia maiores corporis vitae odio tempore totam earum, nam laborum. Nemo sunt cupiditate at earum eius sint hic quod quia cumque amet, ratione obcaecati impedit dolorem eaque recusandae? Voluptatibus repudiandae optio eius, deserunt consequuntur voluptatem nemo aut laborum suscipit cum expedita officiis aliquam recusandae accusantium enim delectus dicta obcaecati autem! Quibusdam aliquam voluptatibus, impedit nobis, cum, et eum nostrum laudantium saepe odio voluptatem veniam nulla! Obcaecati explicabo veritatis quibusdam quo enim, expedita iusto suscipit!</p>
+    </div>
+</body>
+</html>
```
Comment (Pawan1005) [inline Nikita/Assignment_8/ass_8.html:23]: indentation is wrong

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**